bittide-instances/data/tcpspray/start.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# SPDX-FileCopyrightText: 2024 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Start tcpspray on a target IP address.
+tcpspray 100.100.100.100 -e -n 10000
```

Comment:
Could you use something in these ranges?

* 10.0.0.0 to 10.255.255.255
* 172.16.0.0 to 172.31.255.255
* 192.168.0.0 to 192.168.255.255

These ranges are marked for private use.